listenbrainz_spark/stats/user.py

```diff
@@ -131,42 +148,55 @@ def main(app_name):
     Args:
         app_name: Application name to uniquely identify the submitted
                   application amongst other applications
+
+        The script will be run to calculate stats for previous month.
```

Comment:
The docstring is for the main function so referring to it as the script does not make sense. Also, this sentence should be at the top of the docstring.